images/scorecard-test-kuttl/main.go

```diff
@@ -55,16 +54,7 @@ func main() {
 		return
 	}
 
-	var suite *Testsuite
-	if len(jsonReport.Testsuite) == 0 {
-		printErrorStatus(errors.New("empty kuttl test suite was found"))
-		return
-	}
-
-	suite = jsonReport.Testsuite[0]
-
-	s := getTestStatus(suite.Testcase)
-
```

Comment:
Why are we removing this check? My understanding of this check is to:
- Check if there is at least 1 kuttl testsuite defined
- If there are no kuttl testsuites return an error stating that there are no kuttl testsuites for us to run and exit.

I am not opposed to making the error message more descriptive as I agree that `empty kuttl test suite was found` could be a bit misleading. Maybe something like `no kuttl test suites found` would be better to represent that there are no kuttl test suites defined and therefore we can't  run anything.

Reply:
So looking back through this, I think we can still keep this section but do something more like:
```go
if len(jsonReport.Testsuite) == 0 {
	printErrorStatus(fmt.Errorf("no kuttl test suite was found. kuttl may not have run successfully | kuttl logs: %s", getKuttlLogs()))
	return
}
```

This would keep almost all the logic the same while still updating the error message to contain the kuttl logs. Using `printErrorStatus` will also make sure the `TestResult.State` field is set to the `v1alpha3.FailState` value.

Reply:
I've followed your approach with a tiny modification: always store the kuttl logs in the `Log` field in error cases (when `printErrorStatus` is called). Now we have an improved error message and always show the kuttl logs, and everything is nicely formatted :)

Reply:
Sounds perfect! Thanks for doing that!